packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -633,6 +637,62 @@ test.describe('PieInput - Component tests', () => {
         });
     });
 
+    test.describe('Slots', () => {
```

Comment:
I think there should there be a test to check if the slot renders nothing when nothing is passed in? i.e slot: '' or no slot prop at all.